CRM/Upgrade/Incremental/sql/5.29.alpha1.mysql.tpl

```diff
@@ -1,5 +1,9 @@
 {* file to handle db changes in 5.29.alpha1 during upgrade *}
 
+{* https://github.com/civicrm/civicrm-core/pull/17824 *}
+UPDATE civicrm_status_pref SET name = 'checkExtensionsOk' WHERE name = 'extensionsOk' AND domain_id = {$domainID};
```

Comment:
@colemanw do we need to limit it by domain? How will this work on a multidomain instance?

Reply:
@seamuslee001 yes the domain clause is in that query. That's pretty standard for update scripts, as upgrades are performed one domain at a time.

Reply:
@colemanw no i would argue that the upgrade is executed on one domain but it is designed to affect all domains e.g. adding columns affects all domains at once and also the civicrm_domain updates https://github.com/civicrm/civicrm-core/blob/19dc865264f31f29eef39a374d7e3ccc1cb60239/CRM/Upgrade/Form.php#L259 don't affect any single domain it affects them all

Reply:
Honestly I haven't dug too deep into how it works. I know that the `addColumn` stuff has safeguards against running more than once.
And this is what our docs say: https://github.com/civicrm/civicrm-core/blob/bc77d7c0798081f999e6eca92d75fa874fd04e74/CRM/Upgrade/Incremental/sql/README.txt#L11-L30

Reply:
I think @seamuslee001 is right on this - we want to update the data for all domains - it only runs on one but the name has changed for all

Reply:
I can change it, but I'm totally confused now about how upgrades work. If we don't run the upgrade once per domain, then why on earth do so many of our upgrade scripts filter on `{$domainId}`? Here is a search in our `CRM/Upgrade/Incremental` directory:

![image](https://user-images.githubusercontent.com/2874912/87889385-58178000-c9ff-11ea-89c2-c54d34f54c22.png)

Reply:
@colemanw navigation is a per domain-manglable table. Traditionally we have not done a good job with domainn updates on multidomain so it's likely they only run on the upgrade-domain

Reply:
That has certainly been AUG's experience iirc navigation and settings upgrades have generally had to be run again separately for the other domains

Reply:
Ooh, that doesn't sound good at all.
Ok I've removed domain from the where clause in light of those deeper problems.